### DISCUSSION TITLE
Npm no matching version for 3.4.*

### DISCUSSION THREAD

**Discussion by Tjaitil** [tailwindlabs/tailwindcss#13604] [Help] [answered]
Hello,

I noticed earlier that my package.json contained tailwind v 4. For context this project was created with vuejs/create-vue which.

I got a problem with my npm and tried to reinstall version 3.4.2 which I had been using.

However I'm getting this npm error.

` No matching version found for tailwind@^3.4.2.`

I'm slightly confused since tailwind v4 is in alpha from what I can gather from npm, github and google.

Also the registry for tailwind doesn't include anything above version 3.1.0 (https://registry.npmjs.org/tailwind). Anything I'm missing here?

**Comment by wongjn** [comment, marked as answer]
The package name for Tailwind CSS is [`tailwindcss`](https://www.npmjs.com/package/tailwindcss), not [`tailwind`](https://www.npmjs.com/package/tailwind).

**Comment by Tjaitil** [comment]
Oh, what an absolute embarrassment. Thank you.
